projects/common-consumption/src/lib/card/group-card/group-card.component.scss

```diff
@@ -1,6 +1,5 @@
-@import "../../../../../../node_modules/@project-sunbird/sb-styles/assets/mixins/mixins";
-@import "../../../../../../node_modules/@project-sunbird/sb-styles/assets/mixins/custom-mixins";
-@import "../../../../../../node_modules/@project-sunbird/sb-styles/assets/loadingshadows";
+@use "~@project-sunbird/sb-styles/assets/mixins/mixins" as *;
+@use "~@project-sunbird/sb-styles/assets/loadingshadows" as *;
```

Comment:
why do you need two @use ?

Reply:
two @use -> both are different linkings
one is for mixin linking, another one styles linking